.github/workflows/autobuild.yml

```diff
@@ -443,3 +443,89 @@ jobs:
       - name:                       Perform CodeQL Analysis
         if:                         matrix.config.run_codeql
         uses:                       github/codeql-action/analyze@v2
+
+  create_deb_repo:
+    name:                           Create files for .deb repository (if requested)
+    runs-on:                        ubuntu-22.04
+    needs:                          [create_release, release_assets]
+    if:                             needs.create_release.outputs.publish_to_release == 'true'
+    # Set permissions to allow uploading artifact, uploading to release
+    permissions:
+      checks:                       write
+      contents:                     write
+    steps:
+      - name:                       Import GPG key
+        env:
+          GPG_PRIVATE_KEY:          ${{ secrets.GPG_PRIVATE_KEY }}
+        run:                        |
+          set -eu
+
+          [[ "${GPG_PRIVATE_KEY:-}" ]] || {
```

Comment:
Forks which don't have a GPG key set, will have a failed build. 
I'm not sure if a silent failure would be better. Skipping the repo build should be possible.

There are multiple options:
* Remove the whole job (easy)
* Introduce a variable enabling/disabling the repo creation
* If GPG_PRIVATE_KEY is not present, skip the job. We still need to spin up the container, I'm afraid. GitHub dosn't support checking for present secrets in the .yaml file directly as far as I understand

Reply:
Will `needs.create_release.outputs.publish_to_release == 'true'` be true?  Is that what you mean by "failed build"?

Reply:
Yes. If they publish a release it will be true.

Reply:
Ok. Maybe my new approach works.

Reply:
That run demonstrates that skipping the job works if the GPG_PRIVATE_KEY is not set. I can't see a run that demonstrates the action is working properly when the private key IS set.

I'm assuming that the main jamulus repo has a private key, but the ann0see repo does not. But to test this before merging, I guess either ann0see's repo needs a (temporary?) private key, or perhaps we need to merge these changes into a branch on the jamulus repo, so that it can use the jamulus repo secrets?

I don't profess to be an expert (or even moderately knowledgeable) in Debian packaging or repos. Most of my packaging experience is with RPMs on RH-based systems.

Reply:
See https://github.com/ann0see/jamulus/releases/tag/r3_9_1devrepo
That's a working (?) repo